docs/learn-about-neptune/core-concepts.rst

```diff
@@ -1,116 +1,164 @@
+----------------------------------------------------------------------------------------------
 
-Example view, where project Owner can manage project members
+User Roles
+----------
+.. _core-concepts_user-roles:
 
-.. image:: ../_static/images/core-concepts/invite-to-project.png
-   :target: ../_static/images/core-concepts/invite-to-project.png
-   :alt: Invite user to the project
+Subject to their roles, users belong to organizations and collaborate on projects. 
+The roles in an organization are different from the roles in a project.
 
-Public
-^^^^^^
-Public project is freely available to view by everyone who has access to the Internet.
-Examples are: |credit-default-prediction| and |binary-classification-metrics|.
+Roles in an organization
+^^^^^^^^^^^^^^^^^^^^^^^^
+`Have you already created an organization? <team-management.html#how-to-team-management-create-organization>`_
 
-.. External links
+If so, you can invite people to join it. You can assign the members one of two roles: Admin or Member.
 
-.. |credit-default-prediction| raw:: html
+.. _core-concepts_user-roles_organization-admin:
 
-    <a href="https://ui.neptune.ai/neptune-ai/credit-default-prediction" target="_blank">Credit default prediction</a>
+**Admin**
 
+.. note::
 
-.. |binary-classification-metrics| raw:: html
+    An organization must have at least one Admin.
 
-    <a href="https://ui.neptune.ai/neptune-ai/binary-classification-metrics" target="_blank">Binary classification metrics</a>
+By default, an Admin is the Owner of all projects (this setting is editable).
+
+Admins have edit permissions for organization settings, which they can access by 
+clicking **Settings** for the relevant organization.
+
+.. image:: ../_static/images/core-concepts/org-settings.png
+   :target: ../_static/images/core-concepts/org-settings.png
+   :alt: Organization settings button
+   :width: 250
+
+Settings include people and subscription:
+
+.. Also: in the pix below, there are more tabs
+.. Is it Billing or Subscription?
+
+* In the **Subscription** tab, Admins can edit payment options and plans and access invoice data.
+* In the **People** tab, Admins can add people to an organization or remove them.
+
+**Member**
+
+Regular members have no access to organization settings. For projects created in an organization, a member can be 
+assigned by the project Owner.
+
+Roles in a project
+^^^^^^^^^^^^^^^^^^
+
+Members of projects can be one of three types: Owner, Contributor or Viewer.
+
+.. _core-concepts_user-roles_project-owner:
+
+**Owner**
+
+* Has edit access to all experiments and Notebooks.
+* Has edit access to project settings.
+* Can remove projects.
+* A project creator is by default the project Owner.
+
+**Contributor**
+
+* Has edit access to his or her own experiments and Notebooks.
+* Has view access to all his or her experiments and Notebooks.
```

Comment:
```
* Has edit access to his or her own experiments and Notebooks.
* Has view access to all his or her experiments and Notebooks.
```
Contributors have edit access similar to Owners.

Reply:
OK